keyboards/qck75/v1/info.json

```
@@ -0,0 +1,129 @@
+{                                                                                                                         
+    "manufacturer": "Andrian",                                                                                            
+    "keyboard_name": "QCK75-XT",                                                                                          
+    "maintainer": "PoringH",                                                                                              
+    "bootloader": "stm32-dfu",                                                                                            
+    "diode_direction": "COL2ROW",                                                                                         
+    "encoder": {                                                                                                          
+        "rotary": [                                                                                                       
+            {"pin_a": "C0", "pin_b": "C1", "resolution": 2}                                                               
+        ]                                                                                                                 
+    },                                                                                                                    
+    "features": {                                                                                                         
+        "bootmagic": true,                                                                                                
+        "command": false,                                                                                                 
+        "console": false,                                                                                                 
+        "encoder": true,                                                                                                  
+        "extrakey": true,                                                                                                 
+        "mousekey": true                                                                                                  
+    },                                                                                                                    
+    "matrix_pins": {                                                                                                      
+        "cols": ["A6", "A7", "C4", "C5", "B0", "B1", "B2", "B11", "A9", "A8", "C9", "C8", "C7", "C6", "B15", "B14", "B13",
```

Formatting on this file is still invalid, probably down to this missing bracket
```suggestion
        "cols": ["A6", "A7", "C4", "C5", "B0", "B1", "B2", "B11", "A9", "A8", "C9", "C8", "C7", "C6", "B15", "B14", "B13"],
```